src/mainwindow/mainwindow.cpp

```diff
@@ -309,6 +309,12 @@ MainWindow::MainWindow()
         alwaysOnTopAct->setChecked(true);
         setWindowFlags(windowFlags() | Qt::WindowStaysOnTopHint);
     }
+
+    showStatusBarAct->setChecked(getConfig().general.showStatusBar);
+    slot_setShowStatusBar();
```

Comment:
@nschimme Are you ok with calling the slot handler directly? It felt cleaner than duplicating the code to effect the config

Reply:
Yeah, that's fine. They're all functional calls internally if its the same thread anyway :-)